doc/api_methods/AddSubscriber.md

```diff
@@ -0,0 +1,44 @@
+[back to list](../Readme.md)
+
+# Add Subscriber
+
+## `array addSubscriber(array $subsriber [, array $list_ids = [], array $options = []])`
```

Comment:
You didn't mention if this throws any exceptions. For example, does it throw an exception if a passed list doesn't exist?